sympy/integrals/rubi/utility_function.py

```diff
@@ -6678,8 +6680,27 @@ def PolyLog(n, p, z=None):
 def D(f, x):
     return f.diff(x)
 
+def IntegralFreeQ(u):
```

Comment:
I notice that this utility function does not exist in the RUBI files. I would not recommend to introduce new utility functions. SymPy already has its own. Besides, it appears that this utility function is only needed in one place.

Reply:
Sorry, was using an old version of RUBI.